spring-boot-project/spring-boot-docs/src/main/asciidoc/production-ready-features.adoc

```diff
@@ -381,6 +381,9 @@ The following `HealthIndicators` are auto-configured by Spring Boot when appropr
 |{sc-spring-boot-actuator}/mongo/MongoHealthIndicator.{sc-ext}[`MongoHealthIndicator`]
 |Checks that a Mongo database is up.
 
+|{sc-spring-boot-actuator}/neo4j/Neo4jHealthIndicator.{sc-ext}[`Neo4jHealthIndicator`]
+|Checks that a Neo4j server is up.
+
```

Comment:
This is already merged. Please rebase and squash your branch (you can push force on it to update this PR).